packages/client/src/components/Location/LocationFinder.tsx

```diff
@@ -66,7 +70,9 @@ const LocationFinder: FunctionComponent<LocationFinderProps> = ({
 
   const variables = {
     extraHouseNumberFull: "",
-    houseNumberFull: houseNumberFull,
+    houseNumberFull: houseNumberFull
+      ? sanitizeHouseNumberFull(houseNumberFull)
+      : "",
```

Comment:
I would expect returning `""` at `sanitizeHouseNumberFull` when it's not a value. This would clean this up

Reply:
In that case the function should not only expect `string` types. So can you also add a test for:
```
expect(sanitizeHouseNumberFull(undefined)).toBe("");
```